java/src/server/client/SwaggerHandler.java

```diff
@@ -0,0 +1,76 @@
+package server.client;
+
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+import server.games.IServerManager;
+import shared.utils.FileUtils;
+
+import java.io.BufferedOutputStream;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.logging.Logger;
+
+import static server.client.ServerCommunicator.sendResponse;
+
+/**
+ * Handles all the html and json requests to run swagger
+ * Appends ".json" to the request before getting the proper file from the file system
+ */
+public class SwaggerHandler implements HttpHandler {
+    private EndpointDispatcher getMethod, postMethod;
```

Comment:
I really don't think this needs to be here, or anything involving `EndpointDispatcher`.